mosec/coordinator.py

```diff
@@ -72,10 +77,10 @@ def __init__(
                 stage.
             ipc_wrapper (IPCWrapper): IPC wrapper class to be initialized.
         """
-        worker._id = worker_id
+        worker._worker_id = worker_id
         self.worker = worker()
-        self.worker._set_mbs(max_batch_size)
-        self.worker._set_stage(stage)
+        self.worker.max_batch_size = max_batch_size
```

Comment:
maybe use the setter since we already have one, instead of directly assigning?

Reply:
This is using `setter`. The attribute is `_max_batch_size`.